controllers/storagecluster/storagecluster_controller.go

```diff
@@ -7,7 +7,12 @@ import (
 	"reflect"
 
 	"github.com/go-logr/logr"
+	"github.com/google/go-cmp/cmp"
+	"github.com/google/go-cmp/cmp/cmpopts"
```

Comment:
fix the action `cannot find module providing package github.com/google/go-cmp/cmp/cmpopts: import lookup disabled by -mod=vendor`

Reply:
Done